build.gradle

```diff
@@ -320,7 +320,9 @@ dependencies {
   implementation group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.springfoxSwagger
   implementation group: 'org.flywaydb', name: 'flyway-core', version: '8.5.12'
   implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
-  implementation group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: versions.launchDarklySdk
+  implementation group: 'org.yaml', name: 'snakeyaml', version: '1.33'
+
+  implementation group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: '5.10.2'
```

Comment:
As you have a permanent fix, can you please remove these suppressions https://github.com/hmcts/rd-commondata-api/blob/master/config/owasp/suppressions.xml#L34 ?

Reply:
Thanks @SujeethKumarAri removed the suppressions ....